src/plugins/vis_types/timeseries/public/convert_to_lens/lib/configurations/xy/layers.ts

```diff
@@ -170,38 +171,38 @@ const convertAnnotation = (
   annotation: Annotation,
   dataView: DataView
 ): EventAnnotationConfig | undefined => {
-  if (annotation.query_string) {
-    const extraFields = annotation.fields
-      ? annotation.fields
-          ?.replace(/\s/g, '')
-          ?.split(',')
-          .map((field) => {
-            const dataViewField = dataView.getFieldByName(field);
-            return dataViewField && dataViewField.aggregatable ? field : undefined;
-          })
-          .filter(nonNullable)
-      : undefined;
-    return {
-      type: 'query',
-      id: annotation.id,
-      label: 'Event',
-      key: {
-        type: 'point_in_time',
-      },
-      color: new Color(transparentize(annotation.color || euiLightVars.euiColorAccent, 1)).hex(),
-      timeField: annotation.time_field,
-      icon:
-        annotation.icon &&
-        ICON_TYPES_MAP[annotation.icon] &&
-        typeof ICON_TYPES_MAP[annotation.icon] === 'string'
-          ? ICON_TYPES_MAP[annotation.icon]
-          : 'triangle',
-      filter: {
-        type: 'kibana_query',
-        ...annotation.query_string,
-      },
-      extraFields,
-      isHidden: annotation.hidden,
-    };
-  }
+  const extraFields = annotation.fields
+    ?.replace(/\s/g, '')
+    ?.split(',')
+    .map((field) => {
+      const dataViewField = dataView.getFieldByName(field);
+      return dataViewField && dataViewField.aggregatable ? field : undefined;
+    })
+    .filter(nonNullable);
+
+  return {
+    type: 'query',
+    id: annotation.id,
+    label: 'Event',
+    key: {
+      type: 'point_in_time',
+    },
+    color: new Color(transparentize(annotation.color || euiLightVars.euiColorAccent, 1)).hex(),
+    timeField: annotation.time_field || dataView.timeFieldName,
```

Comment:
Isn't the `time_field` a required parameter?

Reply:
no, it's not required for TSVB

Reply:
The I think there's a problem with the label here then:

<img width="343" alt="Screenshot 2022-09-29 at 15 07 38" src="https://user-images.githubusercontent.com/924948/193041082-ab6c3a6e-71f3-49c7-8783-a040a64003c2.png">

If the label is correct, I think the annotations with no `time_field` should just be ignored then in Lens.

Reply:
ok... in fact on the chart I see annotation, so label is definitely not correct 

<img width="881" alt="image" src="https://user-images.githubusercontent.com/20072247/193047049-e6be558d-e299-4258-a62c-6dd046c6fc86.png">

Plus in code I see: 

<img width="818" alt="image" src="https://user-images.githubusercontent.com/20072247/193047521-213bfc64-2b1e-462f-863a-7645aefc507c.png">

Reply:
@dej611  #142226 opened